src/node/desktop/src/main/menu-callback.ts

```diff
@@ -224,31 +234,67 @@ export class MenuCallback extends EventEmitter {
     const accelerator = this.convertShortcut(shortcut);
     template.accelerator = accelerator;
 
-    if (this.setShortcutDebounceId) clearTimeout(this.setShortcutDebounceId);
-
-    this.setShortcutDebounceId = setTimeout(() => {
-      this.updateMenus();
-    }, 5000);
+    this.debounceUpdateMenuLong();
```

Comment:
@timtmok I have removed your code in favor of this simpler one. I kept your 5000 ms timeout, but I was wondering: Does it need to be that long? If a menu update is triggered, we will wait 5 seconds for it to be applied. This function is quite fast when running a single time. Wouldn't a smaller timeout be enough? For app initialization, I found that even 250 would be enough... Unless this is triggered hundreds of times in a 5 seconds time frame... 

Haven't tested this bit to see how many times is this triggered... Any thoughts?

Reply:
This was originally there when I last worked on the menu and I kept the 5000 ms timeout because that's just what was there. 250 ms sounds like it would be enough to have all the shortcut changes updated in the model before updating the Electron menu. 👍